homeassistant/components/logi_circle/strings.json

```diff
@@ -11,16 +11,13 @@
         "description": "Please follow the link below and **Accept** access to your Logi Circle account, then come back and press **Submit** below.\n\n[Link]({authorization_url})"
       }
     },
-    "create_entry": {
-      "default": "Successfully authenticated with Logi Circle."
-    },
     "error": {
-      "auth_error": "API authorization failed.",
-      "auth_timeout": "Authorization timed out when requesting access token.",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "oauth2_authorize_url_timeout": "[%key:common::config_flow::abort::oauth2_authorize_url_timeout%]",
```

Comment:
I know that oauth helper check against key `authorize_url_timeout` but here it's explicitely called by the config flow. I don't know if we have to use helper string or name it like this.

Reply:
I did this to standardize the names  `auth_timeout` -> `oauth2_authorize_url_timeout`
I can not find `authorize_url_timeout ` in `Config flow` `Tests config flow` `strings`

Reply:
```suggestion
      "authorize_url_timeout": "[%key:common::config_flow::abort::oauth2_authorize_url_timeout%]",
```